build/build-and-unit-tests.yml

```diff
@@ -0,0 +1,63 @@
+# This template contains jobs to build and run unit tests
+
+parameters:
+  configuration: ''
+  checkLicenseHeaders: ''
+  runComponentGovernance: ''
+
+jobs:
+- job: BuildAndUnitTests${{ parameters.configuration }}
```

Comment:
>BuildAndUnitTests$ [](start = 7, length = 18)

Should this file reflect that it's only for PR builds? The signed build will need a similar template, but has additional TSA-related steps

Reply:
I agree with you and wonder if we can defer the change until the signed build refactoring. Another reason I mention this is that we can also define sequences of steps (without defining an entire job) that can be reused. This means we could compose files used in both the PR and signed build.

Reply:
Works for me :)